lib/sycamore/sycamore/transforms/partition.py

```diff
@@ -419,6 +419,10 @@ class ArynPartitioner(Partitioner):
              either pdfminer or OCR. Currently supports the 'object_type' property for pdfminer,
              which can be set to 'boxes' or 'lines' to control the granularity of output.
         source: The application that is using the partitioner. This is used for logging purposes.
+        output_label_options: A dictionary for configuring output label behavior. It supports two options: 
+        promote_title, a boolean that specifies whether to add a title to partitioned elements if one is missing, and
+            title_candidate_elements, a list of strings representing labels for potential titles.
+            default: {"promote_title": True ,  "title_candidate_elements":["Section-header", "Caption"]}
```

Comment:
Let's make it more clear that this is only done on the first page.

https://github.com/aryn-ai/sycamore/blob/59f8f2c95eb304365d6857fd756babdc30e22e88/lib/sycamore/sycamore/utils/pdf_utils.py#L171-L172

Reply:
This has been fixed in #1057